var/spack/repos/builtin/packages/gnina/package.py

```diff
@@ -68,13 +68,31 @@ class Gnina(CMakePackage, CudaPackage):
     depends_on("py-numpy", type="build")
     depends_on("py-pytest", type="build")
 
-    depends_on("cuda@11")
+    depends_on("cuda@11", when="@:1.1")
+    depends_on("cuda@12", when="@1.3:")
     depends_on("cudnn", when="+cudnn")
 
+    depends_on("cmake@3.27:", when="@1.3:")  # CMake policy CMP0146 introduced in 3.27
```

Comment:
Is this a requirement? If I understand correctly, in 3.27 and onwards CMake will simply not provide the CUDA module, but older versions already provide the `CUDA` language support. At least this is my understanding based on https://cmake.org/cmake/help/latest/policy/CMP0146.html.

Reply:
Yes, the issue here is that the policy is checked unconditionally. So the fact that doesn't exist triggers a CMake error. (Basically needs something akin to `PR282-v1.3.patch` to work, but I didn't want to add too many patches.

Reply:
Ah, I see... fair enough, then it stays!